### PR TITLE
Pane tests

### PR DESCRIPTION
**Pull request by djbarnwal**
This PR includes test for all the files in `src/components/panes/` except for App info pane (#643 would need to rewrite the component code)

### CODE REVIEW

File: src/components/panes/__tests__/side-pane.test.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+      .toBe(false)
+  })
+
+  it('always renders one Resizable inside Drawer', () => {
+    expect(sidePane().wrap(sidePane().find(Drawer))
+      .find(Resizable)).toHaveLength(1)
+  })
+
+  it("sets the Resizable's onResizeStop prop to correct function", () => {
+    // const changeSidePaneWidth = jest.spyOn(tasks, 'changeSidePaneWidth.callback')
```
Comment (djbarnwal) [inline src/components/panes/__tests__/side-pane.test.js]: What should be the correct approach to test https://github.com/iodide-project/iodide/blob/f5605943d226c040354ee244f7429b0e7c0cbfeb/src/components/panes/side-pane.jsx#L62-L63

File: src/components/panes/__tests__/declared-variables-panes.test.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
+import { shallow } from 'enzyme'
```
Comment (hamilton) [inline src/components/panes/__tests__/declared-variables-panes.test.js]: This file should be `declared-variables-pane.test.js`, no?

File: src/components/panes/__tests__/declared-variables-panes.test.js [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+      },
+    }
+    mountedPane = undefined
+  })
+
+  it('always renders one SidePane', () => {
+    expect(declaredVariablesPane().find(SidePane).length).toBe(1)
+  })
+
+  it("sets the HistoryPane's openOnMode prop to be history", () => {
```
Comment (hamilton) [inline src/components/panes/__tests__/declared-variables-panes.test.js]: copy is wrong on this I think - should be `DeclaredVariablesPane` not `HistoryPane`

File: src/components/panes/__tests__/declared-variables-panes.test.js [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+    expect(declaredVariablesPane().find(SidePane).props().openOnMode)
+      .toBe('declared variables')
+  })
+
+  it("sets the HistoryPane's task prop to be toggleHistoryPane", () => {
```
Comment (hamilton) [inline src/components/panes/__tests__/declared-variables-panes.test.js]: same as above

File: src/components/panes/__tests__/side-pane.test.js (hunk 1 above)
Comment (hamilton) [reply src/components/panes/__tests__/side-pane.test.js]: not sure tbh. @bcolloran?

**Comment by djbarnwal** [comment]
@hamilton I was just about to push the changes for fixing the above issues. Should I address these issues in my next test related PR?

**Pull request merged**